types/identicon.js/index.d.ts

```diff
@@ -105,7 +105,7 @@ export default class Identicon {
         y: number,
         w: number,
         h: number,
-        color: Color,
+        color: string | Color,
```

Comment:
```suggestion
        color: string,
```

For SVGs, this should be a rgba code. For PNGs, this should be a single character encoding a value from the color pallet [as per pnglib](https://github.com/GeorgeChan/pnglib/blob/master/lib/pnglib.js#L119).